src/io/flutter/view/FlutterView.java

```diff
@@ -279,9 +277,65 @@ public void actionPerformed(AnActionEvent event) {
   }
 }
 
+class RepaintRainbowAction extends AbstractToggleableAction {
+  RepaintRainbowAction(@NotNull FlutterView view) {
+    super(view, "Enable Repaint Rainbow");
+  }
+
+  protected void perform(AnActionEvent event) {
+    final Map<String, Object> params = new HashMap<>();
+    params.put("enabled", isSelected(event));
+    view.getFlutterApp().callServiceExtension("ext.flutter.repaintRainbow", params);
+  }
+}
+
+class TimeDilationAction extends AbstractToggleableAction {
+  TimeDilationAction(@NotNull FlutterView view) {
+    super(view, "Enable Slow Animations");
+  }
+
+  protected void perform(AnActionEvent event) {
+    final Map<String, Object> params = new HashMap<>();
+    params.put("timeDilation", isSelected(event) ? 5.0 : 1.0);
+    view.getFlutterApp().callServiceExtension("ext.flutter.timeDilation", params);
```

Comment:
I feel slightly inclined to extract a method since there's a bit of duplicated logic in the perform methods but not a big deal.